include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -474,9 +478,9 @@ struct OrtApi {
   /// @{
 
   /** \brief Create an OrtEnv
-  * 
-  * \param[in] logging_level
-  * \param[in] logid
+  *
+  * \param[in] logging_level The log severity level.
+  * \param[in] logid The top-level log identifier.
```

Comment:
What is the top-level log?

Reply:
There's some hierarchy - this, then a session log id, and then a run log id. Any suggestion for a better way to put it?

Reply:
Well, I guess it's scoped to the environment instance. Won't be more specific here for consistency with other similar parameter descriptions.